util/appdefaults/appdefaults_unix.go

```diff
@@ -9,8 +9,9 @@ import (
 )
 
 const (
-	Address = "unix:///run/buildkit/buildkitd.sock"
-	Root    = "/var/lib/buildkit"
+	Address    = "unix:///run/buildkit/buildkitd.sock"
+	Root       = "/var/lib/buildkit"
+	ConfigPath = "/etc/buildkit/buildkitd.toml"
```

Comment:
Can we have `UserConfigPath`?

Reply:
@AkihiroSuda You mean `/etc/buildkit/buildkitd.toml` for root and `$HOME/.buildkitd.toml` for else or smth different?

Reply:
`$XDG_CONFIG_HOME/buildkit/buildkitd.toml` (`$XDG_CONFIG_HOME` defaults to `~/.config`)